### PR TITLE
 Ad refactoring 3

### PR DESCRIPTION
**Pull request by adumortier**
- added a controller to handle separately the view page for `/items` and for `/merchants/:id/items`
- added an `index` view page for the `merchant_items` controller that only shows `active` items
- changed the `index` view page for the items controller so that it only shows `active` items
- changed the `index` action method for the `items` controller

### CODE REVIEW

File: app/views/items/index.html.erb [hunk 1]
```diff
@@ -1,27 +1,18 @@
-<% if @merchant %>
-  <h1><%= link_to @merchant.name, "/merchants/#{@merchant.id}"%><span> Items</span></h1>
-  <p align="center"><%= link_to "Add New Item", "/merchants/#{@merchant.id}/items/new" %></p>
-<% else %>
+
   <h1>All Items</h1>
-<% end %>
 <section class="grid-container">
   <% @items.each do |item| %>
+  <% if item.active? %>
```
Comment (cgaddis36) [inline app/views/items/index.html.erb:5]: I like it thank you

File: spec/features/items/index_spec.rb [hunk 2]
```diff
@@ -56,5 +53,218 @@
...
+      end
+
+      @tire.update(active?: false)
+
+      visit '/items'
+
+      expect(page).to_not have_link(@tire.name)
+      expect(page).to_not have_content(@tire.description)
+      expect(page).to_not have_css("img[src*='#{@tire.image}']")
+
```
Comment (cgaddis36) [inline spec/features/items/index_spec.rb:128]: Nice testing here thank you for fixing this

File: spec/features/items/index_spec.rb [hunk 3]
```diff
@@ -56,5 +53,218 @@
...
+        expect(page).to have_link(@dog_bone.name)
+        expect(page).to have_content(@dog_bone.description)
+        expect(page).to have_content("Price: $#{@dog_bone.price}")
+        expect(page).to have_content("Inventory: #{@dog_bone.inventory}")
+        expect(page).to have_link(@brian.name)
+        expect(page).to have_css("img[src*='#{@dog_bone.image}']")
+      end
+
+    end
+
```
Comment (cgaddis36) [inline spec/features/items/index_spec.rb:268]: nice job testing each type of user

**Review by cgaddis36 — APPROVED** [review]

**Review by KellyIB — APPROVED** [review]
tests look great.

**Pull request merged**